docs/quickstart.rst

```diff
@@ -26,6 +26,32 @@ Compute a Keccak256 Hash
   >>> keccak(b'')
   b"\xc5\xd2F\x01\x86\xf7#<\x92~}\xb2\xdc\xc7\x03\xc0\xe5\x00\xb6S\xca\x82';{\xfa\xd8\x04]\x85\xa4p"
 
+
+You may also compute hashes incrementally
+
+.. doctest::
+
+  >>> from eth_hash.auto import keccak
+  >>> digest = keccak.Digest(b'part-a')
+  >>> digest.update(b'part-b')
+  >>> digest.digest()
```

Comment:
I like the [`pycryptodome` API](https://www.pycryptodome.org/en/latest/src/hash/hash.html#api-principles) more, the only difference being that it opens with `new()`.

```py
preimage = keccak.new(b'part-a')
preimage.update(b'part-b')
preimage.digest()
```